guides/common/modules/proc_attaching-satellite-infrastructure-subscription.adoc

```diff
@@ -3,18 +3,18 @@
 
 [NOTE]
 ====
-This step is *optional* if you have SCA enabled in the Red Hat Customer Portal.
+This step is *optional* if you have SCA enabled on {Project}.
```

Comment:
Like I said in the bz, I feel like the word "optional" is misleading, since with SCA enabled sub-man won't even _let_ you attach a subscription.

Reply:
Reworded :+1: